rest/src/main/kotlin/builder/interaction/MultiApplicationCommandBuilder.kt

```diff
@@ -6,25 +6,46 @@ import kotlin.contracts.InvocationKind
 import kotlin.contracts.contract
 
 @KordDsl
-public class MultiApplicationCommandBuilder {
+public sealed class MultiApplicationCommandBuilder {
     public val commands: MutableList<ApplicationCommandCreateBuilder> = mutableListOf()
 
-    public inline fun message(name: String, builder: MessageCommandCreateBuilder.() -> Unit = {}) {
+    public fun build(): List<ApplicationCommandCreateRequest> {
+        return commands.map { it.toRequest() }
+    }
+}
+
```

Comment:
```suggestion
public inline fun MultiApplicationCommandBuilder.input(
    name: String,
    description: String,
    builder: ChatInputCreateBuilder.() -> Unit = {},
) {
    contract { callsInPlace(builder, InvocationKind.EXACTLY_ONCE) }
    commands += ChatInputCreateBuilderImpl(name, description).apply(builder)
}

public inline fun MultiApplicationCommandBuilder.message(
    name: String,
    builder: MessageCommandCreateBuilder.() -> Unit = {},
) {
    contract { callsInPlace(builder, InvocationKind.EXACTLY_ONCE) }
    commands += MessageCommandCreateBuilderImpl(name).apply(builder)
}


public inline fun MultiApplicationCommandBuilder.user(name: String, builder: UserCommandCreateBuilder.() -> Unit = {}) {
    contract { callsInPlace(builder, InvocationKind.EXACTLY_ONCE) }
    commands += UserCommandCreateBuilderImpl(name).apply(builder)
}

```
this is what I was referring to in this comment: https://github.com/kordlib/kord/pull/604#discussion_r867287095
the same could be done for the command behaviors